package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "presto-js-client",
+  "version": "0.0.0",
+  "license": "MIT",
```

Comment:
I saw apache 2.0 license file :-p

Reply:
haha this was generated by Nx, good catch, will fix